From: ismael@sourcemage.org

musl libc does not implement _SC_LONG_BIT, replace with a portable alternative.

--- a/src/core/abi.cc
+++ b/src/core/abi.cc
@@ -20,7 +20,7 @@ __ID("@(#) $Id: mem.cc 1352 2006-05-27 23:54:13Z ezix $");
 bool scan_abi(hwNode & system)
 {
   // are we compiled as 32- or 64-bit process ?
-  system.setWidth(sysconf(LONG_BIT));
+  system.setWidth(sizeof(long) * CHAR_BIT);
 
   pushd(PROC_SYS);
 
